upgrade.php

```diff
@@ -244,7 +244,7 @@ function getSqlServer()
     //test if mysql or mariadb is used.
     $sql = 'SELECT VERSION() as version';
     $result = $link->query($sql);
-    if (strpos($result->fetch_array['version'], 'MariaDB') === false) {
+    if (strpos($result->fetch_array()['version'], 'MariaDB') === false) {
```

Comment:
both our codes are valid (direct echo vs variable assignment and echo variable), but substance doesn't change: on php 5.3 you can't access the array returned by a method directly, you need to assign the method result to a temp variable and then use that temp variable (it's array dereferencing, introduced in PHP 5.4)